### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by jimlynnjulian** [MrS0m30n3/youtube-dl-gui#498]
Hello,

If the computer crashes, is there a copy of the url list in the registry, or elsewhere, that will allow me to continue?

Thank you

**Comment by axcore** [comment]
youtube-dl-gui is not being maintained any more. There are various forks around, including [this one](https://github.com/axcore/tartube).

In that fork, click the **Classic Mode** tab, click the menu button in the top-right corner, and select **Remember URLs**.

**Comment by jimlynnjulian** [comment]
Thank you axcore.

I just installed Tartube x64. 
The startup file is not an executable so I cannot place the icon on the toolbar at the bottom of the desktop.
The System Preferences/Windows/System Tray form has two checkboxes:
Show icon in system tray
Close to tray, rather than closing the application
I checked both boxes, but after closing using the 'power' button, the icon disappears from the system tray after a few seconds.
Is restarting the computer required to get this update to the preferences?
